Apps/Src/SendTritium.c

```diff
@@ -638,29 +645,25 @@ void Task_SendTritium(void *p_arg){
     state = FSM[NEUTRAL_DRIVE];
     prevState = FSM[NEUTRAL_DRIVE];
 
-    #ifndef __TEST_SENDTRITIUM
-    CANbus_Init(MOTORCAN);
+    CANbus_Init(MOTORCAN, NULL, 0);
     CANDATA_t driveCmd = {
         .ID=MOTOR_DRIVE, 
         .idx=0,
         .data={0.0f, 0.0f},
     };
-    #endif
 
     while(1){
         prevState = state;
 
         state.stateHandler();    // do what the current state does
-        #ifndef __TEST_SENDTRITIUM
         readInputs();   // read inputs from the system
         UpdateDisplay_SetAccel(accelPedalPercent);
-        #endif
         state.stateDecider();    // decide what the next state is
 
         // Drive
-        #ifdef __TEST_SENDTRITIUM
+        #ifdef DEBUG
         dumpInfo();
-        #else
+        #endif
         if(MOTOR_MSG_COUNTER_THRESHOLD == motorMsgCounter){
             memcpy(&driveCmd.data[0], &currentSetpoint, sizeof(float));
```

Comment:
These should be flipped. No need to change it here (the fix is in M2-testing) but I just want to make sure that the change doesn't disappear during a merge, so keep it in mind if you have to merge in master